src/OpenConext/EngineBlockApiClientBundle/Service/AttributeReleasePolicyService.php

```diff
@@ -65,10 +65,10 @@ public function applyAttributeReleasePolicies(ConsentList $consentList, Attribut
             $definitions[$name] = $attribute->getAttributeDefinition();
         }
 
-        $response = $this->jsonApiClient->post('/arp', [
+        $response = $this->jsonApiClient->post([
             'entityIds'  => $entityIds,
-            'attributes' => $mappedAttributes
-        ]);
+            'attributes' => !empty($mappedAttributes) ? $mappedAttributes : new stdClass()
+        ], '/arp');
```

Comment:
Just for readability's sake, could you put the data in a variable and then `post($data, '/arp')`?